scrapers/SissyPov.yml

```diff
@@ -0,0 +1,34 @@
+name: SissyPov
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - sissypov.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="videoDetails clear"]/h3
+      Performers:
+        Name: //li[@class="update_models"]/a
+      Date:
+         selector: //div[@class="videoInfo clear"]/p[1]
+         postProcess:
+           - replace:
+               - regex: 'Date Added:\s*'
+                 with: ""
+           - parseDate: January 2, 2006
+      Tags:
+        Name: //div[@class="featuring clear"]//li/a
```

Comment:
the above selector includes performers
```yaml
        Name: //ul[contains(li,"Tags:")]//a
```
contains only the tags
Or you could filter yours so that only the `a` with `/categories/` in the `@href` are matched